ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/DataStructureUtil.java

```diff
@@ -813,6 +813,16 @@ public Attestation randomAttestation() {
             this::randomCommitteeBitvector);
   }
 
+  public Attestation randomElectraAttestation() {
+    return getElectraSchemaDefinitions(randomSlot())
+        .getAttestationSchema()
+        .create(
+            randomBitlist(),
+            randomAttestationData(),
+            randomSignature(),
+            this::randomCommitteeBitvector);
+  }
+
```

Comment:
i commented above, but would prefer this did not exist.

Reply:
Removed it since the DataStructure util uses the spec it's been initialised with so there is no need for this in the end